app/views/proprietor/accounts/_add_data_for_cross_tenant_search.html.erb

```diff
@@ -0,0 +1,25 @@
+
+
+<p>
+  <%= f.check_box :shared_search, {checked: account.shared_search_tenant?, name: "account[settings][shared_search]", id: "account_settings_shared_search"}, true , false %>
```

Comment:
Is Rubocop happy with this line?
```suggestion
  <%= f.check_box :shared_search, { checked: account.shared_search_tenant?, name: "account[settings][shared_search]", id: "account_settings_shared_search"}, true , false %>
```

Also, I personally like creating helper methods for these, even if they are for one time use. Leaves the views much cleaner, which is an issue with ERB:
```
<%= enable_shared_search_check_box(f) %>
```
and then in the helper:

```
def enable_shared_search_check_box(f, account)
    f.check_box :shared_search, {checked: account.shared_search_tenant?, name: "account[settings][shared_search]", id: "account_settings_shared_search"}, true , false
end
```

Reply:
Yes Ruboocp was happy hence no rubocop failures. I will move it to a helper.